### ISSUE TITLE
Text alternatives that are not alternatives confusion

### ISSUE THREAD

**Issue by LaurenceRLewis** [w3c/wcag#951]
Could someone explain what the following paragraph in the document 'Failure of Success Criterion 1.1.1 and 1.2.1 due to using text alternatives that are not alternatives (e.g., filenames or placeholder text) actually means please. [https://www.w3.org/WAI/WCAG21/Techniques/failures/F30.html](url)

`This describes a failure condition for all techniques involving text alternatives. If the text in the "text alternative" cannot be used in place of the non-text content without losing information or function then it fails because it is not, in fact, an alternative to the non-text content.`

Does this apply to <input> form fields that:

- Have only placeholder text and no other visible label (fail for 3.3.2 Labels or Instructions)
- Have placeholder text and a visible label (not programatically associated with the input) i.e. no for attribute or other method. _(does this fail any forms related SC?). 

I’m sure the latter fails 1.3.1 Info and Relationships, however don’t think it would fail 3.3.2 Labels or Instructions
 
Thanks for any feedback

**Comment by JAWS-test** [comment]
I think this is not about form fields, but about graphics, audio files and videos that don't have a meaningful alternative text. Examples for such alternative texts are given in F30.

For me, the sentence says the following: all information that the graphic (video etc.) visually communicates must also be available in the text alternative.

**Comment by LaurenceRLewis** [comment]
> all information that the graphic (video etc.) visually communicates must also be available in the text alternative.

That certainly makes more sense than the current sentence. 

Thanks @JAWS-test

**Comment by alastc** [comment]
The technique is linked from SC 1.1.1 (and 1.2.1), which are for images and multimedia. The technique doesn't mention forms, so I'm not sure there is a useful update to make.

If the explanation from @JAWS-test works for you, can we close this issue @LaurenceRLewis ?
